components/TARGET_PSA/inc/psa/storage_common.h

```diff
@@ -33,8 +33,10 @@ extern "C" {
  */
 typedef uint32_t psa_storage_create_flags_t;
 
-#define PSA_STORAGE_FLAG_NONE        0         /**< No flags to pass */
-#define PSA_STORAGE_FLAG_WRITE_ONCE (1 << 0) /**< The data associated with the uid will not be able to be modified or deleted. Intended to be used to set bits in `psa_storage_create_flags_t`*/
+#define PSA_STORAGE_FLAG_NONE                 0         /**< No flags to pass */
+#define PSA_STORAGE_FLAG_WRITE_ONCE           (1 << 0)  /**< The data associated with the uid will not be able to be modified or deleted. Intended to be used to set bits in `psa_storage_create_flags_t`*/
+#define PSA_STORAGE_FLAG_NO_CONFIDENTIALITY   (1 << 1)  /**< The data associated with the uid is public and therefore does not require confidentiality. It therefore only needs to be integrity protected */
```

Comment:
According to the spec linked, this should be `PSA_STORAGE_FLAG_INTEGRITY_ONLY`, not `PSA_STORAGE_FLAG_NO_CONFIDENTIALITY`.

Reply:
Please look at the up to date version of [storage_common.h in master](https://github.com/ARMmbed/psa_trusted_storage_api/blob/master/include/psa/storage_common.h). It holds the `PSA_STORAGE_FLAG_NO_CONFIDENTIALITY ` value.

Reply:
Could we ask for a release tag in psa_trusted_storage_api? The latest release, `Release-PSA1.0RC2` looks out of date.

Then, we can reference the fresh tag from this PR's commits so it's clear which version of the PSA Storage API Mbed OS implements.

Reply:
Asked for it in there.